common/network-common/src/test/java/org/apache/spark/network/sasl/SparkSaslSuite.java

```diff
@@ -374,6 +378,71 @@ public void testDelegates() throws Exception {
     }
   }
 
+  @Test
+  public void testSaslEncryptionAes() throws Exception {
+    System.setProperty(BLOCK_SIZE_CONF, "10k");
```

Comment:
There are exceptions that can be thrown before these properties are cleaned up. Can you try to avoid using system properties for this?